packages/edit-site/src/components/sidebar/global-styles-v2/screens/typography-element.js

```diff
@@ -0,0 +1,286 @@
+/**
+ * External dependencies
+ */
+import { isUndefined, startCase } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Button,
+	ButtonGroup,
+	CardBody,
+	Dropdown,
+	DropdownMenu,
+	DropdownMenuItem,
+	DropdownTrigger,
+	FormGroup,
+	Grid,
+	ListGroup,
+	Select,
+	Slider,
+	TextInput,
+	UnitInput,
+	useNavigatorParams,
+	VStack,
+} from '@wordpress/components';
+import {
+	formatCapitalize,
+	formatLowercase,
+	formatUppercase,
+} from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { Screen, ScreenHeader } from '../components';
+import { useAppState } from '../state';
+
+const typographyOptions = [
```

Comment:
Seems like this belongs with the global styles state and not specifically with this screen?